<launch>
  <arg name="cloud_in" default="/cloud_in"/>
  <arg name="cloud_out" default="/cloud_out"/>

  <node name="pointcloud_filter" pkg="squirrel_pointcloud_filter" type="squirrel_pointcloud_filter_node"
        output="screen" respawn="true">
    <remap from="/cloud_in" to="$(arg cloud_in)"/>
    <remap from="/cloud_out" to="$(arg cloud_out)"/>
    <rosparam file="$(find squirrel_pointcloud_filter)/config/asus_filter.yaml" command="load" />
  </node>

</launch>
